cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -230,6 +230,63 @@ def single_qubit_randomized_benchmarking(
     return RandomizedBenchMarkResult(num_clifford_range, gnd_probs)
 
 
+def parallel_single_qubit_randomized_benchmarking(
```

Comment:
OOC, would it be possible to put this in the existing `single_qubit_randomized_benchmarking` module? Seems like it's basically the same thing and there could be opportunities for code reuse. The fact that one is running in parallel on multiple qubits is a minor details (the existing single-qubit code could just become a special case of the parallel code).

Reply:
Ok, I changed it so that `single_qubit_randomized_benchmarking` is now a wrapper for `parallel_single_qubit_randomized_benchmarking`. What do you think?

Reply:
I changed it so that `single_qubit_randomized_benchmarking` is a wrapper for a special case of `parallel_single_qubit_randomized_benchmarking` . What do you think?